src/imageloader.js

```diff
@@ -262,6 +272,17 @@ function completeJob(loader, job, callback) {
         loader.jobsInProgress++;
     }
 
+    if (loader.tileRetryMax > 0 && loader.jobQueue.length === 0) {
+        //SAME AS ABOVE => REFACTOR
```

Comment:
@paaddyy Do you remember what you meant by this?

Reply:
just saw your mention here, sorry.. no, I don't know what I meant with this comment, it's too long ago 😄

Reply:
Oh well, thanks for looking!

Reply:
I think here he meant that this if could be refactored to follow the above if. However, in the above one there is no setTimeout, so I guess it's fine having it like this.

Reply:
Makes sense :)